apps/rush-lib/package.json

```diff
@@ -46,7 +46,8 @@
     "tar": "~5.0.5",
```

Comment:
Can we also support creating `.tar`/`.tgz` archives?

Reply:
In the future, but IMO there's no urgency to implement that now, since the current design accomodates it.